test/unit/runner.spec.js

```diff
@@ -499,29 +512,47 @@ describe('Runner', function() {
     });
 
     describe('hugeStackTrace', function() {
-      beforeEach(function() {
-        if (path.sep !== '/') {
+      before(function() {
+        if ('browser' in process) {
           this.skip();
         }
       });
 
-      it('should not hang if the error message is ridiculously long single line', function(done) {
+      // Generate 64k string
```

Comment:
@cyjake, did you determine the lower bound of what you termed "ridiculously long"?